transforms/universal/rep_removal/dpk_rep_removal/runtime.py

```diff
@@ -85,7 +85,7 @@ def add_input_params(self, parser: ArgumentParser) -> None:
             "--rep_removal_num_threads",
             type=str,
```

Comment:
Would it make sense to make this int ?
```suggestion
            type=int,
```